app/assets/javascripts/application.js

```diff
@@ -14,4 +14,4 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
-//= require bootstrap-sprockets
\ No newline at end of file
+= require bootstrap-sprockets
```

Comment:
Oh ok! When I was running the app locally I was getting an error saying that `bootstrap-sprockets` was missing. I'll put it back to how it was and re-push